test/bdd/fixtures/did-keys/publickeys.json

```diff
@@ -0,0 +1,26 @@
+[
+ {
+  "id": "key1",
+  "type": "Ed25519VerificationKey2018",
+  "encoding": "Jwk",
```

Comment:
Is there a reason why not to use publicKeyJwk?

Reply:
done